spec/RateLimit.spec.js

```diff
@@ -481,6 +481,14 @@ describe('rate limit', () => {
     expect(() =>
       validateRateLimit({ rateLimit: [{ requestTimeWindow: 3, requestCount: 'abc' }] })
     ).toThrow('rateLimit.requestPath must be defined');
+    expect(() =>
+      validateRateLimit({ rateLimit: [{ requestPath: '/*' }] })
+    ).toThrow(`rateLimit.requestPath "/*" uses deprecated wildcard syntax. ` +
+      `Please update to path-to-regexp v8 syntax. Examples:\n` +
+      `  Old: "/functions/*" → New: "/functions/*path"\n` +
+      `  Old: "/classes/*" → New: "/classes/*path"\n` +
+      `  Old: "*" → New: "*path"\n` +
+      `See: https://github.com/pillarjs/path-to-regexp#usage`);
```

Comment:
Does the new syntax have a name? So we don't have to link to a 3rd party site? The examples etc should go into the migration guide.